### ISSUE TITLE
flowspec: AND bit in the first operator is set

### ISSUE THREAD

**Issue by iwaseyusuke** [Exa-Networks/exabgp#772]
##### ISSUE TYPE

Bug Report

##### OS

ExaBGP is running on Ubuntu 16.04 Docker container on Ubuntu 16.04 host.

##### VERSION

```
# exabgp --version
ExaBGP : 4.0.2-1c737d99
Python : 3.5.2 (default, Nov 23 2017, 16:37:01)  [GCC 5.4.0 20160609]
Uname  : Linux 46616c3d145e 4.4.0-109-generic #132-Ubuntu SMP Tue Jan 9 19:52:39 UTC 2018 x86_64
Root   : /usr/local
```

##### ENVIRONMENT 

```
<EMPTY>
```

##### CONFIGURATION

```
neighbor 172.17.0.2 {
    router-id 192.168.0.2;
    local-address 172.17.0.3;
    local-as 65000;
    peer-as 65000;
}
```

##### SUMMARY

With the combined numeric type (maybe also bitmask type) rules like `port >=80&<=90`, ExaBGP seems to set the AND bit of in the first operator byte.
Wireshark shows `&& >=80 && <=90`;

```
...(snip)...
Border Gateway Protocol - UPDATE Message
    ...(snip)...
    Path attributes
        ...(snip)...
        Path Attribute - MP_REACH_NLRI
            ...(snip)...
            Network layer reachability information (6 bytes)
                FLOW_SPEC_NLRI (6 bytes)
                    ...(snip)...
                    Filter: Port filter (&& >=80 && <=90)
                    ...(snip)...
```

According to [RFC5575](https://tools.ietf.org/html/rfc5575), it should be unset, then the above rule should be `>=80 && <=90` on Wireshark.

```
         a -   AND bit.  If unset, the previous term is logically ORed
               with the current one.  If set, the operation is a logical
               AND.  It should be unset in the first operator byte of a
               sequence.  The AND operator has higher priority than OR
               for the purposes of evaluating logical expressions.
```

##### STEPS TO REPRODUCE

After connection established, announce a route via "exabgpcli".

```
exabgpcli 'announce flow route { match { port >=80&<=90; } then { discard; } }'
```

**Comment by thomas-mangin** [comment]
Thank you for reporting this problem.

**Comment by thomas-mangin** [comment]
Hello @iwaseyusuke - exabgp own decoding was also showing the extra '&' and this is now fixed - thank you for being more thorough than I am with testing ...

**Comment by thomas-mangin** [comment]
Hello @iwaseyusuke something went wrong with the pypi release of 4.0.3 and forgot to fix it -- pypi did some change and it prevented my automation to work !
now released 4.0.5 just for you 😜

**Comment by iwaseyusuke** [comment]
@thomas-mangin Great! I will use it!
